### ISSUE TITLE
As an owner or chapter admin I can cancel events and all participants who have RSVP'd will get an email notification.

### ISSUE THREAD

**Issue by QuincyLarson** [freeCodeCamp/chapter#100]
The UI exists and emails get sent, but we still need text for the email.

Depends on https://github.com/freeCodeCamp/chapter/pull/814

Once that is done, we should lock this down to the approved roles and confirm via Cypress tests

- [x] As an owner I can cancel any event
- [x] As an admin of a Chapter I can cancel any events in that Chapter
- [x] As any other role, I cannot cancel events 
- [x] Sensible email content
- [ ] update the test for it

The main thing to check is that requests from unauthorised users fail, but it's also helpful to confirm that the UI reflects their role. 

---

Test should account for developer typing logic, that stop canceling the event if there are no members in the event Reference https://github.com/freeCodeCamp/chapter/pull/1517#discussion_r1029353250
Test should account for the mutation deletion reference https://github.com/freeCodeCamp/chapter/pull/1517#discussion_r1029353250

**Comment by allella** [comment]
Linking these for when someone starts working on email sending.

#283 #146 are related to the transactional email needed to send over SMTP.

Also, some questions which may be relevant were raised in https://github.com/freeCodeCamp/chapter/issues/320#issuecomment-581653505

**Comment by allella** [comment]
This would apply to other bulk email sending as well, but the current default shows / exposes multiple recipients email addresses in the To: line in Mailhog.

We'd obviously want a BCC: for any bulk email sending.

It was suggested in https://github.com/freeCodeCamp/chapter/pull/909#issuecomment-1046295697 to 

> automatically handled by MailerService - depending if submitted email list has more than one entries. But either way that's for another PR.

**Comment by Ismailtlem** [comment]
Hi,

Is this issue only about changing the body text of the email that get sent when cancelling an event?

**Comment by allella** [comment]
I think @gikf has worked on a lot of the interface side, so I'm pinging and directing that question accordingly.

**Comment by gikf** [comment]
Yep, from more immediate things - I believe so.

I think test for it might need changing later, once seeding will seed also unsubscribed users.

**Comment by Sboonny** [comment]
Is there an email content decided? Or do I alter another email content for now, and we can think of one post MVP?

**Comment by ojeytonwilliams** [comment]
@Sboonny is the test you mentioned updated?  If it is, could you close this?

**Comment by Sboonny** [comment]
Nope, but I think we are done with the MVP part, we can open new issue for test.

**Comment by ojeytonwilliams** [comment]
Okay, cool.  Could you take care of that? I think you know best what still needs to be done.

**Comment by Sboonny** [comment]
Of course, I wanted to go over it again 👍

**Comment by ojeytonwilliams** [comment]
Great, thanks for looking into this. Canceling events without members seems to be working at the moment, so this can be closed.
